features/get_task.feature

```diff
@@ -2,4 +2,7 @@ Feature: Get a task
 
   Scenario: User gets a task
     TODO: Add steps for getting a task
-    
\ No newline at end of file
+  Given the user is using the task manager application for looking at or getting a task by name
```

Comment:
Hi zusammen,
ich finde die initiale Version eurer Feature Definition einen guten Start. Ich würde allerdings auf eine klare Beschreibung der Ausganssituation achten. Aktuell bennent ihr zwei Bedigungen: (i) "for looking at" und (ii) "or getting a task by name". Ich würde hier nur einen Fall berücksichtigen, was die Umsetzung vereinfacht.

"Given the user is using the task manager application and whats to receive information of an existing task by name"